fulltime-developer.md

```diff
@@ -38,33 +38,59 @@ On the side:
 
 ## Who you are
 
-- You care about web standards
-- You're fluent in HTML, CSS and Javascript
-- You love the web for its openness, and build websites accordingly, available for everyone on every device in every - browser
-- You can breakdown a complex issue into smaller parts to solve
+You will be working in a team and you will have to opportunity to learn and grow.  
+We do not expect you to have mastered every aspect we value but we do expect some expertise in key areas.
+
+Please let us know your strengths and weaknesses, we're looking for a good fit, not a perfect match.
+
+Do not hesitate to apply if you're not sure you meet all the requirements.  
+We are open to all candidates and will consider all applications.
+
+### Technical skills
+- You're fluent in HTML and CSS
+- You have experience with WordPress (or PHP)
+- You have experience with accessibility and are familiar with WCAG
+- You have some experience with JavaScript (not that important)
+
+### Technical skills (bonus)
+- You can create new interactive components that are fully accessible using WCAG examples as a guide
+- You have in-depth knowledge of WordPress
+- You know how a page is rendered, how scripts initiate and how styles are applied at a fundamental level
+- You have experience with handlebars-like templates (we use Twig)
+- You can write highly maintainable code
+- You can write easily reviewable code
+- You can manage your own focus points and balance your work between hard problems and routine tasks
+- You are aware of "Thinking, Fast and Slow" and can leverage it to ease your work
+
+### Soft skills
+- You ask for help
```

Comment:
```suggestion
- You can ask for help
```

Reply:
This phrasing makes it seem that they need to be skilled at how they ask for help or when they ask for help.

While the basic skill is simply doing the asking.

I prefer to keep the original